### DISCUSSION TITLE
Is it secure enough to use also internet accessable?

### DISCUSSION THREAD

**Discussion by carlchen2007** [ciur/papermerge#552] [Q&A]
Can i use ClamAV for Login?(if yes how can I integrate)
What could be a problem for security?(are there any known flaws?)
And is there any security benefit of using docker instead of bare metal?

**Comment by ciur** [comment]
I cannot answer your question as I don't know what is ClamAV.

**Reply by carlchen2007** [reply]
I am sorry i meant Fail2Ban...
